pkgs/development/python-modules/qiskit/default.nix

```diff
@@ -8,22 +8,36 @@
 , qiskit-ibmq-provider
 , qiskit-ignis
 , qiskit-terra
+  # Optional inputs
+, withOptionalPackages ? false
+, qiskit-finance ? null
+, qiskit-machine-learning ? null
+, qiskit-nature ? null
+, qiskit-optimization ? null
```

Comment:
```suggestion
, withOptionalPackages ? false
, qiskit-finance
, qiskit-machine-learning
, qiskit-nature
, qiskit-optimization
```

Reply:
These packages (-finance, -ml, -nature, -optimization) aren't in nixpkgs yet, though they are available in my NUR https://github.com/drewrisinger/nur-packages. I was trying to leave them out of this PR to make it easier to review, and add them in later. I can remove the optional packages for now if you prefer.

Reply:
I guess it makes sense to leave them out cause it's almost as easy to override `propagatedBuildInputs`.

Reply:
Removed these optional inputs.